src/adapters/MeetingsJSONAdapter.js

```diff
@@ -854,4 +886,61 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
 
     return concat(initialState$, settingsEvent$);
   }
+
+  /**
+   * Returns an observable that emits the display data of the video switcher control.
+   *
+   * @param {string} ID  ID of the meeting for which to update display
+   * @returns {Observable.<MeetingControlDisplay>} Observable that emits control display data
+   * @private
+   */
+  videoControl(ID) {
+    const meeting = this.datasource[ID];
+    const videoDevices = defer(() => this.getAvailableDevices('videoinput'));
+    let initialState$;
+
+    if (meeting) {
+      initialState$ = videoDevices.pipe(
+        map((devices) => ({
+          ID: SWITCH_VIDEO_DEVICE,
+          tooltip: 'Video Devices',
+          options: devices && devices.map((device) => ({
+            value: device.deviceId,
+            label: device.label,
+            device,
+          })),
+          selected: meeting.videoDeviceID,
```

Comment:
I suspect `meeting.videoDeviceID` is not set initially, so this would be set to `undefined`. We'll revisit that.